docs/index.rst

```diff
@@ -110,53 +194,94 @@ A note on encoding detection
 If your input is a mess of unmarked bytes, you might want a tool that can just
 statistically analyze those bytes and predict what encoding they're in.
 
-ftfy is not that tool. The `guess_bytes` function will do this in very limited
-cases, but to support more encodings from around the world, something more is
-needed.
+`ftfy` is not that tool. The :func:`ftfy.guess_bytes` function it contains will
+do this in very limited cases, but to support more encodings from around the
+world, something more is needed.
 
-You may have heard of chardet. Chardet is admirable, but it doesn't completely
-do the job either. Its heuristics are designed for multi-byte encodings, such
-as UTF-8 and the language-specific encodings used in East Asian languages. It
-works badly on single-byte encodings, to the point where it will output wrong
-answers with high confidence.
+You may have heard of `chardet`. Chardet is admirable, but it doesn't
+completely do the job either. Its heuristics are designed for multi-byte
+encodings, such as UTF-8 and the language-specific encodings used in East Asian
+languages. It works badly on single-byte encodings, to the point where it will
+output wrong answers with high confidence.
 
-ftfy's `guess_bytes` doesn't even try the East Asian encodings, so the ideal thing
-would combine the simple heuristic of `guess_bytes` with the multibyte character
-set detection of `chardet`. This ideal thing doesn't exist yet.
+:func:`ftfy.guess_bytes` doesn't even try the East Asian encodings, so the
+ideal thing would combine the simple heuristic of :func:`ftfy.guess_bytes` with
+the multibyte character set detection of `chardet`. This ideal thing doesn't
+exist yet.
 
 
 Accuracy
 --------
 .. include:: accuracy.rst
 
+
+Command-line tool
+-----------------
+ftfy can be used from the command line. By default, it takes UTF-8 input and
+writes it to UTF-8 output, fixing problems in its Unicode as it goes.
+
+Here's the usage documentation for the `ftfy` command::
```

Comment:
This is out of date now (there's a `-o` option and the help text for `--preserve-entities` has changed).  There's no way to auto-generate this, is there?